src/main/haskell/kore/src/Kore/Step/Step.hs

```diff
@@ -118,7 +120,12 @@ transitionRule tools substitutionSimplifier simplifier =
             $ stepWithAxiom tools substitutionSimplifier config a
         case result of
             Left _ -> pure []
-            Right (config', proof') ->
+            Right
+                ( StepResult
+                    { rewrittenPattern = config'
+                    -- TODO(virgil): Also use the remainder
+                    }
+                , proof') ->
                 if ExpandedPattern.isBottom config'
```

Comment:
Since we are using `ExpandedPattern.isBottom` here, did we need to normalize `\bottom` patterns above?